webrender/src/clip_scroll_node.rs

```diff
@@ -412,6 +411,38 @@ impl ClipScrollNode {
             _ => false,
         }
     }
+
+    pub fn find_unclipped_rectangle(&self,
```

Comment:
It appears to be similar to the `MashBounds::inner`. Perhaps, you could just use `clip_info.mask_cache_info.bounds.inner` instead of having a separate method?